scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/RunProcessor.java

```diff
@@ -127,25 +126,32 @@ public static ObjectMapper createObjectMapper() {
     return mapper;
   }
 
+  /** Attempt to parse naively, and force parsing using UTF-8 if that doesn't work */
   public static Optional<Document> parseXml(File file) {
     try {
-      DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+      return Optional.of(DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(file));
+    } catch (ParserConfigurationException | IOException | SAXException e) {
+      log.error("Not really a UTF-16 parsing exception, forcing UTF-8 parsing...", e);
+      DocumentBuilder builder;
+      try {
+        builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+      } catch (ParserConfigurationException e2) {
+        throw new RuntimeException(e2);
+      }
       // Automatically detect BOMs and remove them from input stream
       // BOM characters can interfere with text processing if not properly removed
       try (BOMInputStream bomInputStream = new BOMInputStream(new FileInputStream(file));
           Reader reader = new InputStreamReader(bomInputStream, StandardCharsets.UTF_8)) {
         return Optional.of(builder.parse(new InputSource(reader)));
-      } catch (FileNotFoundException | SAXException e) {
-        log.error("Failed to parse XML", e);
+      } catch (SAXException | IOException e3) {
+        log.error("Failed to parse XML", e3);
```

Comment:
```suggestion
        log.error("Failed to parse XML after forcing UTF-8 encoding", e3);
```

Having a different message will help with debugging